docs/Advanced-Topics.rst

```diff
@@ -77,3 +77,26 @@ Recommendations for gcc Users (MinGW, \*nix)
 --------------------------------------------
 
 -  Refer to `gcc Tips <./gcc-Tips.rst>`__.
+
+Support for Position Bias Treatment
+--------------------------------------------
+
+Often the relevance labels provided in Learning-to-Rank taks might be derived from implicit user feedback (e.g., clicks) and therefore might be biased due to their position on the screen when having been presented to a user. LightGBM uses an additional file to store positional data, like the following:
+
+::
+
+    3
+    4
+    1
+    ...
+
+It means the position of the document from the first data row, when presented to a user, was ``3``, second was ``4``, and so on. A position could be any (arbitrary) integer number and could encode the presentation of an item even in complex layouts (it is up to the LightGBM user to perform such encoding/enumeration).
+
+The position file corresponds with training data file line by line, and has one position per line.
+
+And if the name of training data file is ``train.txt``, the position file should be named as ``train.txt.position`` and placed in the same folder as the data file.
```

Comment:
@metpavel @shiyu1994 why are we keeping this restriction? I think it's quite limiting, unnecessarily.

I thought that in https://github.com/microsoft/LightGBM/pull/5929/files#r1257685144, we'd agreed to instead provide a path to a file through user-provided configuration in `params`.

Reply:
Done via 
102fb97

Reply:
Update document in Advanced Topics to remove this constraint via [757f7cb](https://github.com/microsoft/LightGBM/pull/5929/commits/757f7cb5b19134394d47dec792b29cfb3b225223)